modules/ROOT/pages/users.adoc

```diff
@@ -60,7 +62,25 @@ By default, in every new organization and business group, you get three options:
 Roles and permissions are grouped under organizations (and also, optionally, under business groups). This means that you can only assign roles and permissions that are related to resources that exist in the organization and/or business group that you are selecting.
 --
 
-If necessary, you can also remove user permissions, but keep in mind that if users have been granted their permission through an assigned role, you cannot view or remove those role-based permissions here--you must do it using the xref:roles.adoc[Roles] section.
+If necessary, you can also remove user permissions, but note that if users have been granted their permissions through an assigned role, you can remove role-based permissions using only the xref:roles.adoc[Roles] functionality. You cannot view or remove role-based permissions in the *Users* page.
+
+Because the Roles feature is deprecated, it is best to xref:teams.adoc#opt-in-to-teams[opt in to the Teams feature] to manage permissions in a hierarchy.
```

Comment:
Maybe the entire topic should begin with this: why discuss the ins and outs of deprecated features, if the audience can't or won't be using those features?

Reply:
The most probable use-case is that someone will be migrating from roles to teams over the next few months incrementally. As far as the IA for this repo goes, it does not make sense to disambiguate the roles deprecation in this topic.